easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.5-CUDA-9.1.85.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '7.0.5'
+cuda_version = '9.1.85'
+
+cuda_version_major_minor = '.'.join(cuda_version.split('.')[:2])
+
+versionsuffix = '-CUDA-%s' % cuda_version
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+# Download as cudnn-9.0-linux-x64-v7.tgz, rename to cudnn-9.0-linux-x64-v7.0.2.tgz
```

Comment:
fix filename to rename to (`7.0.5`, not `7.0.2`), or make it generic: `rename to cudnn-9.0-linux-x64-v<VERSION>.tgz`